crates/sequencer/src/runner.rs

```diff
@@ -72,7 +74,7 @@ where
 {
     da_service: Arc<Da>,
     mempool: Arc<CitreaMempool<C>>,
-    sov_tx_signer_priv_key: C::PrivateKey,
+    sov_tx_signer_priv_key: Vec<u8>,
```

Comment:
```suggestion
    sov_tx_signer_priv_key: k256::SecretKey,
```

Please try to reduce usage of raw Vec<u8> in favor of typed entities

Reply:
I keep it as vec because I will need to convert it Dalek as well for backwards compatibility

Reply:
Ok. Please add todo to change its type before we release mainnet. And create an issue

Reply:
https://github.com/chainwayxyz/citrea/issues/1874